ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractMessageProcessor.java

```diff
@@ -155,8 +153,6 @@ protected void completedFailed(final MessageFrame frame) {
   private void codeExecute(final MessageFrame frame, final OperationTracer operationTracer) {
     try {
       evm.runToHalt(frame, operationTracer);
-    } catch (final ExceptionalHaltException e) {
-      frame.setState(MessageFrame.State.EXCEPTIONAL_HALT);
```

Comment:
It seems like this ended up being redundant, is that right?

Reply:
Yea, exceptional halts are no longer directly signaled via Java exceptions.